### ISSUE TITLE
[FE][SPIKE] What does using LoROTA with Clipboard look like?

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#26515]
Answer that question^?

**Comment by mdewey** [comment]
Idea: 

Skip manager to fill out form, and manager because more historic

**Comment by mdewey** [comment]
What is the minimal validation needs, no design changes with LoROTA

**Comment by mdewey** [comment]
We have an engineer meeting set for July 13th 2021 for the engineering team to sync up on the details and early questions

Full doc here: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/questionnaire/engineering/LoROTA

**Comment by mdewey** [comment]
We wargamed out some solutions, my notes are located https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/engineering/LoROTA

TL;DR; Backburner priority, possible but needs some edge cases though out still
